src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberFieldSymbol.cs

```diff
@@ -188,7 +188,6 @@ internal static DeclarationModifiers MakeModifiers(NamedTypeSymbol containingTyp
                 Debug.Assert((result & ~(DeclarationModifiers.AccessibilityMask | DeclarationModifiers.Fixed | DeclarationModifiers.Unsafe | DeclarationModifiers.New)) == 0);
             }
 
-
             if ((result & DeclarationModifiers.Const) != 0)
```

Comment:
It doesn't look like there are meaningful changes in this file, consider reverting.